src/groups/mwc/mwcu/mwcu_blobiterator.h

```diff
@@ -44,11 +43,16 @@ namespace mwcu {
 // ==================
 
 /// A blob iterator
-class BlobIterator : public bsl::iterator<bsl::forward_iterator_tag,
```

Comment:
Deprecated
https://en.cppreference.com/w/cpp/iterator/iterator

The intended way now is to define public iterator typedefs manually in the class declaration

Reply:
Update: removed iterator typedefs because we don't actually need it